tests/sanity/ignore-2.10.txt

```diff
@@ -945,7 +945,6 @@ plugins/modules/identity/keycloak/keycloak_clienttemplate.py validate-modules:pa
 plugins/modules/identity/onepassword_info.py validate-modules:parameter-list-no-elements
 plugins/modules/identity/opendj/opendj_backendprop.py validate-modules:doc-missing-type
 plugins/modules/identity/opendj/opendj_backendprop.py validate-modules:parameter-type-not-in-doc
-plugins/modules/monitoring/airbrake_deployment.py validate-modules:doc-default-does-not-match-spec
 plugins/modules/monitoring/airbrake_deployment.py validate-modules:doc-missing-type
```

Comment:
Would you mind fixing this one as well? It is about missing `type:` in option docs.

Reply:
Done